PolysyncLidarTranslator/README.md

```diff
@@ -0,0 +1,62 @@
+To properly build this example with the generated PolySync messages in ROS, you will first need to set up your catkin workspace with the ros_bridge.
+
+If you do not have a catkin workspace already, follow these steps to set one up.
+
+http://wiki.ros.org/catkin/Tutorials/create_a_workspace
+
+If you do have a workspace feel free to continue with the steps below. First source your ROS install and then create a package in the src directory of your catkin workspace as shown below.
+
+    $ cd ~/catkin_ws/src
+    $ source /opt/ros/indigo/setup.bash
+    $ catkin_create_pkg polysync_lidar_translator std_msgs roscpp tf
+
+Copy "polysync_lidar_translator_node.cpp" to the src directory of the polysync_lidar_translator package you created.
+
+    $ cp PolySync-Core-CPP-Examples/PolysyncLidarTranslator/polysync_lidar_translator_node.cpp ~/catkin_ws/src/polysync_lidar_translator/src/
+
+Make sure that in the "Build" section of the CMakeLists file for your PolysyncLidarTranslator project are uncommented. All other lines in the "Build" section should be commented.
+
+    ## Declare a C++ executable
```

Comment:
They actually don't strangely, the tab over causes it to be displayed properly as a code block if you look at it with the github gui. However, I will change this to make it more explicit.